routeid/routeid.go

```diff
@@ -0,0 +1,29 @@
+package routeid
```

Comment:
it's used in both the innkeeper and etcd packages. I created a new package in order not to create circular dependencies between packages.